src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -31,7 +31,7 @@
 $should_display = $event->dates->start_display->format( 'Y-m-d' ) === $day_date
                   || $is_start_of_week;
 
-$classes = get_post_class( [ 'tribe-events-calendar-month__multiday-event' ], $event->ID );
+$classes = tribe_get_post_class( [ 'tribe-events-calendar-month__multiday-event' ], $event->ID );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 44b4dad62a199198d5ebe4732fc99c56</sub>

Reply:
Ignoring.